<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>worksPeriodDialog</class>
 <widget class="QDialog" name="worksPeriodDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Работы за период</string>
  </property>
  <widget class="QPushButton" name="m_closeButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>640</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_sumTimeLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>И того: 000:00</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_setTimeButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Задать период и вывести  результаты</string>
   </property>
   <property name="text">
    <string>Показать</string>
   </property>
  </widget>
  <widget class="QTableView" name="m_reportView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>881</width>
     <height>461</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>601</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Начало периода:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="m_dateFromEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Завершение периода:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="m_dateToEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
